scss/_variables.scss

```diff
@@ -646,13 +646,13 @@ $outline-offset:              $outline-width !default; // Deprecated in v5.2.3
 $focus-visible-zindex:                5 !default; // Boosted mod
 
 $focus-visible-inner-width:           2px !default; // Boosted mod
-$focus-visible-inner-color:           $white !default; // Boosted mod
-$focus-visible-inner-color-inverted:  $black !default; // Boosted mod
+$focus-visible-inner-color:           var(--#{$prefix}body-bg) !default; // Boosted mod
```

Comment:
Not a big fan of using a "background" color to define a text color. If no other custom vars are possible, we can let it as is for now. (several occurrences in this PR)

Reply:
Modification to do later, tracked in https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/1776